docs/public/sdk/Metrics.md

```diff
@@ -0,0 +1,53 @@
+# Metrics
+
+Then show how to add each of the 4 instruments available
+
+- Counter
+- ObservableCounter
+- Histogram
+- Gauge
+
+Extract common code related to adding metrics to a program.
+
+## Setup
+
+1. `Counter`
```

Comment:
?

Reply:
I only included a setup for `Counter` for now and once we feel good about the format and how to publish these docs in a more user friendly format I can add the others. @reyang also just changed the metrics example, so I could replace this with the new version if there are no other concerns with this PR 🙂

Reply:
I see two options for how we can reformat it.

1. We can add a descriptive sentence like: `The following steps demonstrate how to create a counter.`
1. We can change it to a header like: `## Create a counter`